drivers.go

```diff
@@ -4,6 +4,9 @@ import (
 	"strings"
 
 	"github.com/docker/libnetwork/driverapi"
+	"github.com/docker/libnetwork/ipamapi"
+	builtinIpam "github.com/docker/libnetwork/ipams/builtin"
```

Comment:
ok. since we will not be exposing the driver package structure in any API, am fine with doing it in a different PR.